src/site/xdoc/cmdline.xml.vm

```diff
@@ -450,6 +450,355 @@ java -D&lt;property&gt;=&lt;value&gt;  \
+      <p><strong>Goal:</strong> Generate an XML file containing suppressions for all violations.</p>
+
+      <p><strong>Command:</strong></p>
+       <div class="wrapper"><pre class="prettyprint"><code class="language-bash">
+      $ java -jar checkstyle-10.18.1-all.jar -c config.xml -g -o suppressions.xml Test.java
+      </code></pre></div>
+
+      <p><strong>Output (inside suppressions.xml):</strong></p>
+       <div class="wrapper"><pre class="prettyprint"><code class="language-xml">
+    &lt;?xml version="1.0" encoding="UTF-8"?&gt;
+    &lt;!DOCTYPE suppressions PUBLIC "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+        "https://checkstyle.org/dtds/suppressions_1_1.dtd"&gt;
+    &lt;suppressions&gt;
+        &lt;suppress files="Test.java" checks="FallThrough"/&gt;
+    &lt;/suppressions&gt;
+    </code></pre></div>
+
+      <h3>7. Setting Tab Width (-w, --tabWidth)</h3>
+      <p><strong>Goal:</strong> Set the length of the tab character when using the `-s` option.</p>
+
+      <p><strong>Command:</strong></p>
+       <div class="wrapper"><pre class="prettyprint"><code class="language-bash">
+      $ java -jar checkstyle-10.18.1-all.jar -s 9:9 --tabWidth 4 Test.java
+      </code></pre></div>
+
+      <p><strong>Output:</strong></p>
+       <div class="wrapper"><pre class="prettyprint"><code class="language-xml">
+      Suppression XPath query:
+      &lt;suppress xpath="//CASE_GROUP" files="Test.java"/&gt;
+    </code></pre></div>
+
+      <h3>8. Displaying Abstract Syntax Tree (-t, --tree)</h3>
+      <p><strong>Goal:</strong> Display the Abstract Syntax Tree (AST) without comments.</p>
+
+      <p><strong>Command:</strong></p>
+       <div class="wrapper"><pre class="prettyprint"><code class="language-bash">
+      $ java -jar checkstyle-10.18.1-all.jar --tree Test.java
+      </code></pre></div>
+
+      <p><strong>Output:</strong></p>
+       <div class="wrapper"><pre class="prettyprint"><code class="language-bash">
+      CLASS_DEF -> Test [1:0]
+       `--METHOD_DEF -> foo [2:2]
+           `--SLIST -> { [3:22]
+               |--VARIABLE_DEF -> int i [4:10]
+               |--WHILE -> while [5:10]
+               `--SWITCH -> switch [6:14]
+      </code></pre></div>
+
+      <h3>9. Displaying AST with Comments (-T, --treeWithComments)</h3>
+      <p><strong>Goal:</strong> Display the AST including comment nodes, but excluding Javadoc.</p>
+
+      <p><strong>Command:</strong></p>
+       <div class="wrapper"><pre class="prettyprint"><code class="language-bash">
+      $ java -jar checkstyle-10.18.1-all.jar --treeWithComments Test.java
+      </code></pre></div>
+
+      <p><strong>Output:</strong></p>
+       <div class="wrapper"><pre class="prettyprint"><code class="language-bash">
+      CLASS_DEF -> Test [1:0]
+       `--METHOD_DEF -> foo [2:2]
+           `--SLIST -> { [3:22]
+               |--VARIABLE_DEF -> int i [4:10]
+               |--WHILE -> while [5:10]
+               |--BLOCK_COMMENT_BEGIN -> /* [8:10]
```

Comment:
Let's update Test.java to have multi line comment, as small javadoc over method.
And we should have here whole content of output as is.

Reply:
If you have special Test.java for special example, please print again, whole county.

Please print whole content of output for all examples